src/pharmpy/model/statements.py

```diff
@@ -814,7 +877,7 @@ def get_compartment_inflows(self, compartment):
         >>> from pharmpy.modeling import load_example_model
         >>> model = load_example_model("pheno")
         >>> model.statements.ode_system.get_compartment_inflows(output)
-        [(Compartment(CENTRAL, amount=A_CENTRAL, dose=Bolus(AMT, admid=1)), CL/V)]
+        [(Compartment(CENTRAL, amount=A_CENTRAL, dose=Bolus(AMT, admid=2)), CL/V)]
```

Comment:
Why do we have `admid=2` here? Only one route should mean `admid=1` as default?

Reply:
`admid=1` is defined to represent an oral dose and `amid=2` represents an IV dose. Since this is a one-compartment system with a Bolus dose directly into the CENTRAL compartment, I believe `admid=2` should be the correct numbering. Or am I missing something?

Reply:
Yes, but only for multiple doses. If there is only one dose it should be `admid=1`. It is only an id and we should really have a way of defining which is which, but until we have that in place I think this makes the most sense.

Reply:
ok, i'll correct this to only change the admid in the presence of multiple doses